examples/IRrecvDumpV2/platformio.ini

```diff
@@ -50,3 +50,8 @@ build_flags = -D_IR_LOCALE_=it-IT  ;  Italian
 
 [env:zh-CN]
 build_flags = -D_IR_LOCALE_=zh-CN  ;  Chinese (Simplified)
+
+; Build the library with all protocols disabled to flush out #if/#ifdef issues &
+; any compiler warnings, by turning them into errors.
+[env:no-protocols]
```

Comment:
Same here, maybe use same name as in SmartIRRepeater/platformio.ini  to be consistent?

Reply:
Ditto. Fixed.